docs/REST/authentication.md

```diff
@@ -461,3 +461,73 @@ def sign_request(host,
         "Authorization": "HMAC-SHA256 Credential=" + credential + "&SignedHeaders=" + signed_headers + "&Signature=" + signature
     }
 ```
+### PowerShell
+```PowerShell
+function Sign-Request(
+    [string] $serverHost,
```

Comment:
how about `$hostname`. serverHost host sounds a bit strange.

Reply:
Thanks, I renamed the param